### PR TITLE
Mps mistral lora

### PR DESCRIPTION
**Pull request by maximegmd**
Additional MPS example to train a Mistral Lora.

Some documentation on usage and limitations.

### CODE REVIEW

**Review by winglian — APPROVED** [review]

File: examples/mistral/lora-mps.yml [hunk 1]
```diff
@@ -0,0 +1,83 @@
+base_model: mistralai/Mistral-7B-v0.1
+model_type: MistralForCausalLM
+tokenizer_type: LlamaTokenizer
+is_mistral_derived_model: true
```
Comment (NanoCode012) [inline examples/mistral/lora-mps.yml]:
```suggestion
```

Would recommend removing this config as it's redundant.

File: examples/mistral/lora-mps.yml [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+saves_per_epoch: 1
+debug:
+deepspeed:
+weight_decay: 0.0
+fsdp:
+fsdp_config:
+special_tokens:
+  bos_token: "<s>"
+  eos_token: "</s>"
+  unk_token: "<unk>"
```
Comment (NanoCode012) [inline examples/mistral/lora-mps.yml]:
```suggestion
special_tokens:
```

Same here. It's not necessary as this is the default special tokens.

File: README.md [hunk 3]
```diff
@@ -95,17 +95,30 @@ Get started with Axolotl in just a few steps! This quickstart guide will walk yo
 
 **Requirements**: Python >=3.9 and Pytorch >=2.0.
 
+General case:
 `pip3 install "axolotl[flash-attn,deepspeed] @ git+https://github.com/OpenAccess-AI-Collective/axolotl"`
 
+Mac:
+`pip3 install "axolotl @ git+https://github.com/OpenAccess-AI-Collective/axolotl"`
+
```
Comment (NanoCode012) [inline README.md:101]: Would you be able to add a link to the new Mac doc you've created also?

File: README.md [hunk 4]
```diff
@@ -95,17 +95,30 @@ Get started with Axolotl in just a few steps! This quickstart guide will walk yo
 
 **Requirements**: Python >=3.9 and Pytorch >=2.0.
 
+General case:
 `pip3 install "axolotl[flash-attn,deepspeed] @ git+https://github.com/OpenAccess-AI-Collective/axolotl"`
 
+Mac:
```
Comment (winglian) [inline README.md]:
```suggestion
Mac: see https://github.com/OpenAccess-AI-Collective/axolotl/blob/13199f678b9aab39e92961323bdbce3234ee4b2b/docs/mac.md
```

**Pull request merged**